src/app/donation-status.type.ts

```diff
@@ -1,3 +1,7 @@
+/**
+ * For details of status semantics see PHP enum MatchBot\Domain\DonationStatus at
+ * https://github.com/thebiggive/matchbot/blob/main/src/Domain/DonationStatus.php
```

Comment:
This link is dead right now because the PHP Enum hasn't been merged to main - but https://github.com/thebiggive/matchbot/blob/develop/src/Domain/DonationStatus.php has the enum and there's an open PR to add a docblock to every case.